config/generic_domains.php

```diff
@@ -33,3 +33,826 @@
+    '.*.com',
+    '.*.com',
+    '.community',
+    '.company',
+    '.computer',
+    '.condos',
+    '.construction',
+    '.consulting',
+    '.contractors',
+    '.cooking',
+    '.cool',
+    '.coop',
+    '.country',
+    '.courses',
+    '.cr',
+    '.credit',
+    '.creditcard',
+    '.cricket',
+    '.cruises',
+    '.cuisinella',
+    '.cx',
+    '.cymru',
+    '.cz',
+    '.dad',
+    '.dance',
+    '.dating',
+    '.datsun',
+    '.day',
+    '.dclk',
+    '.de',
+    '.deals',
+    '.degree',
+    '.delivery',
+    '.democrat',
+    '.dental',
+    '.dentist',
+    '.desi',
+    '.design',
+    '.dev',
+    '.dev',
+    '.diamonds',
+    '.diet',
+    '.digital',
+    '.direct',
+    '.directory',
+    '.discount',
+    '.dk',
+    '.dm',
+    '.dm',
+    '.do',
+    '.docs',
+    '.domains',
+    '.doosan',
+    '.durban',
+    '.dvag',
+    '.dz',
+    '.earth',
+    '.eat',
+    '.ec',
+    '.*.ec',
+    '.edu',
+    '.edu',
+    '.edu',
+    '.education',
+    '.ee',
+    '.email',
+    '.emerck',
+    '.energy',
+    '.engineer',
+    '.engineering',
+    '.enterprises',
+    '.epson',
+    '.equipment',
+    '.es',
+    '.esq',
+    '.estate',
+    '.eu',
+    '.eurovision',
+    '.eus',
+    '.eus',
+    '.events',
+    '.exchange',
+    '.expert',
+    '.exposed',
+    '.fail',
+    '.fans',
+    '.farm',
+    '.fashion',
+    '.feedback',
+    '.fi',
+    '.fi',
+    '.film',
+    '.finance',
+    '.financial',
+    '.firmdale',
+    '.fish',
+    '.fishing',
+    '.fit',
+    '.fitness',
+    '.fj',
+    '.flights',
+    '.florist',
+    '.flowers',
+    '.fly',
+    '.fm',
+    '.fo',
+    '.foo',
+    '.football',
+    '.forsale',
+    '.foundation',
+    '.fr',
+    '.frl',
+    '.frogans',
+    '.fun',
+    '.fund',
+    '.furniture',
+    '.futbol',
+    '.ga',
+    '.gal',
+    '.gallery',
+    '.garden',
+    '.gb',
+    '.gbiz',
+    '.gd',
+    '.gdn',
+    '.gent',
+    '.gf',
+    '.gg',
+    '.ggee',
+    '.gi',
+    '.gift',
+    '.gifts',
+    '.gives',
+    '.gl',
+    '.glass',
+    '.gle',
+    '.global',
+    '.global',
+    '.globo',
+    '.gmail',
+    '.gmx',
+    '.goldpoint',
+    '.goo',
+    '.goog',
+    '.google',
+    '.gop',
+    '.gov',
+    '.gov',
+    '.gq',
+    '.graphics',
+    '.gratis',
+    '.green',
+    '.gripe',
+    '.gs',
+    '.guide',
+    '.guitars',
+    '.guru',
+    '.gy',
+    '.hamburg',
+    '.hangout',
+    '.haus',
+    '.healthcare',
+    '.help',
+    '.here',
+    '.hiphop',
+    '.hiv',
+    '.hk',
+    '.hk',
+    '.*.hk',
+    '.hm',
+    '.hn',
+    '.hn',
+    '.holdings',
+    '.holiday',
+    '.homes',
+    '.horse',
+    '.host',
+    '.host',
+    '.hosting',
+    '.house',
+    '.how',
+    '.hr',
+    '.ht',
+    '.hu',
+    '.ibm',
+    '.icu',
+    '.id',
+    '.*.id',
+    '.ie',
+    '.ifm',
+    '.il',
+    '.im',
+    '.immo',
+    '.immobilien',
+    '.in',
+    '.in',
+    '.inc',
+    '.industries',
+    '.infiniti',
+    '.info',
+    '.info',
```

Comment:
This TLD is duplicated, but it could be changed to ".info.*" to support domains like ".info.pa".